src/ai-bundle/templates/data_collector.html.twig

```diff
@@ -235,11 +296,11 @@
                     </tr>
                     <tr>
                         <th>Arguments</th>
-                        <td>{{ dump(call.call.arguments) }}</td>
+                        <td><pre>{{ dump(call.call.arguments) }}</pre></td>
```

Comment:
can we have the real dump, including the colors an being able to open close nodes inside the dump?